doc/installers.rst

```diff
@@ -54,3 +54,44 @@ they double-click on the scripts in the installation directory, the working
 directory will be the installation directory. Your application shouldn't
 rely on having a particular working directory; if it does, use :func:`os.chdir`
 to set it first.
+
+Packaging with tkinter
+----------------------
+
+Because Pynsist makes use of the "bundled" versions of Python the ``tkinter``
+module isn't included by default. If your application relies on ``tkinter`` for
+a GUI then you need to find the following assets:
+
+* The ``tcl`` directory in the root directory of the version of Python you are
+  using in your app. This must be the same bitness and version.
```

Comment:
We should say somewhere that these files need to come from a Windows install of Python, since the idea of Pynsist is generally that you can build from Linux/Mac as well, and this is a case where that doesn't easily work.